cornavirus.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Novel Coronavirus 2020
+description: Data and analysis on the Coronavirus Pandemic
+keywords: Coronavirus, Covid-19
+date: 2020-03-21
+modified: 2020-03-21
+image: 
+---
+
+Qu: What are the key figures of Covid-19 worldwise, such as:
+- total number of people infected,
+- total number of people recovered,
+- and total number of people that died.
+
+An: Novel Coronavirus 2019 time series case data for confirmed, recovered and deaths - https://datahub.io/core/covid-19
+
+---
+
+## README
+
+The [Severe acute respiratory syndrome Coronavirus 2](https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2) (SARS-CoV-2) has had a worldwide effect. The World Health Organization (WHO) on March 11 declared COVID-19 ([Coronavirus disease 2019](https://en.wikipedia.org/wiki/Coronavirus_disease_2019)) a pandemic, pointing to the over 118,000 cases of the coronavirus illness in over 110 countries and territories around the world at the time. 
+
+This dataset provides time series data tracking the number of people affected by COVID-19 worldwide, including:
```

Comment:
This page is the collections page - not the single dataset - so it should be framed in that way.